packages/edit-site/src/store/test/selectors.js

```diff
@@ -1,34 +1,9 @@
-/**
- * WordPress dependencies
- */
-import { store as coreDataStore } from '@wordpress/core-data';
-
 /**
  * Internal dependencies
  */
-import {
-	getCanUserCreateMedia,
-	getEditedPostType,
-	getEditedPostId,
-	isPage,
-} from '../selectors';
+import { getEditedPostType, getEditedPostId, isPage } from '../selectors';
 
 describe( 'selectors', () => {
-	const canUser = jest.fn( () => true );
-	getCanUserCreateMedia.registry = {
-		select: jest.fn( () => ( { canUser } ) ),
-	};
-
-	describe( 'getCanUserCreateMedia', () => {
-		it( "selects `canUser( 'create', 'media' )` from the core store", () => {
-			expect( getCanUserCreateMedia() ).toBe( true );
-			expect(
-				getCanUserCreateMedia.registry.select
-			).toHaveBeenCalledWith( coreDataStore );
-			expect( canUser ).toHaveBeenCalledWith( 'create', 'media' );
```

Comment:
I guess alternatively we could expect the deprecation message.

Reply:
Do you find similar tests useful?

Reply:
Nope, I personally don't like these pointless unit tests, just pointing it out :D